.github/workflows/build-toolchain.yml

```diff
@@ -50,30 +50,28 @@ jobs:
           path: build\${{ inputs.preset }}\_deps
           key: cmake-deps-${{ inputs.preset }}-${{ hashFiles('CMakePresets.json','cmake/**/*.cmake','**/CMakeLists.txt') }}
 
-      - name: Download VC6 Portable from Cloudflare R2
+      - name: Download VC6 Portable from itsmattkc repo
         if: ${{ startsWith(inputs.preset, 'vc6') && steps.cache-vc6.outputs.cache-hit != 'true' }}
         env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.R2_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.R2_SECRET_ACCESS_KEY }}
-          AWS_ENDPOINT_URL: ${{ secrets.R2_ENDPOINT_URL }}
-          EXPECTED_HASH: "118D0F1ACBBD70C3F8B081CA4DBAF955FE0C6C359A76636E930AA89FDC551091"
+          EXPECTED_HASH: "5FA2FB0FE61FD0E0FE08DC2C77EA33DF9947A39ED13670DD14B30DA5997F9315"
         shell: pwsh
         run: |
           Write-Host "Downloading VC6 Portable Installation" -ForegroundColor Cyan
-          aws s3 cp s3://github-ci/VS6_VisualStudio6.7z VS6_VisualStudio6.7z --endpoint-url $env:AWS_ENDPOINT_URL
+          Invoke-WebRequest -Uri https://github.com/itsmattkc/MSVC600/archive/refs/heads/master.zip -OutFile VS6_VisualStudio6.zip
```

Comment:
`https://github.com/itsmattkc/MSVC600/archive/refs/heads/master.zip`

Shouldn't this refer to a fixed hash instead of head?